### ISSUE TITLE
Type checker for deploy scripts

### ISSUE THREAD

**Issue by nmushegian** [nexusdev/dapple#173]
```
// contract FeedBase is MakerUser
var feedbase = new FeedBase();
```

This does not cause an error, but it should, since FeedBase has a constructor argument. This may be worth addressing before a complete type checker is added.

**Comment by mhhf** [comment]
This is solved in #179 with a hacky solution: the length of the arguments given to a constructor must match, otherwise an error is thrown.
This should be revisited with a full type checking solution as soon as there is capacity.

**Comment by mhhf** [comment]
Resolved in 0.8 since DappleScript == Solidity, solc will throw an Error
